app/ante/ante.go

```diff
@@ -45,6 +45,9 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 				case "/ethermint.types.v1.ExtensionOptionsWeb3Tx":
 					// handle as normal Cosmos SDK tx, except signature is checked for EIP712 representation
 					anteHandler = newCosmosAnteHandlerEip712(options)
+				case "/ethermint.evm.v1.ExtensionOptionDynamicFeeTx":
```

Comment:
the problem with using a tx extension is that I don't think wallets like Keplr  / Cosmostation will use it

Reply:
yeah, it's a matter of standards, there was a discussion before of providing a feemarket module in cosmos-sdk, but we didn't reach a consensus.
Maybe better to make the `max_prio_price` default to max int, the default behavior would be more intuitive, the user pay full `gas_price`, and gets priority proportional to `gas_price - base_fee`.

Reply:
correct, but we'd need the post tx handler for that (as I mentioned in the other issue) to get the refund after the tx is processed

Reply:
opened issue: https://github.com/evmos/ethermint/issues/1231